src/mixins/CartesianChartPropsMixin.js

```diff
@@ -27,12 +27,11 @@ module.exports =  {
     xAxisFormatter:    React.PropTypes.func,
     legend:            React.PropTypes.bool,
     legendOffset:      React.PropTypes.number,
-    width:             React.PropTypes.number,
-    height:            React.PropTypes.number,
+    width:             React.PropTypes.node,
+    height:            React.PropTypes.node,
```

Comment:
I think you want `React.PropTypes.oneOfType([React.PropTypes.number, React.PropTypes.string])` here.